client/src/components/Header/Menu.tsx

```diff
@@ -31,6 +32,9 @@ interface Props extends WithStyles<typeof styles> {
 }
 
 function Menu({ classes, currentPathname, onLoginClicked }: Props) {
+  const { user } = useContext(UserContext);
+  const authenticated = user._id !== '0';
```

Comment:
Hmmm maybe we should make `user` possibly undefined or null in the `UserContext` and null/undefined in the initial state. Is there any reason it's populated when the user isn't signed in?

Reply:
I'm 100% down for that actually. That would make our lives a lot easier. Will implement

Reply:
updated in 48b9d10776c25b5fc24a8411c69180091fed98bf